teamsupport/utils.py

```diff
@@ -0,0 +1,22 @@
+from lxml import etree
+
+
+def to_xml(root, data=None, cdata=None):
+    xml = etree.Element(root)
+
+    if data:
+        for k, v in data.iteritems():
+            child = etree.SubElement(xml, k)
+            child.text = v
+
+    if cdata:
```

Comment:
I took this code from the flow we had for creating XML to send to HelpSpot where we created CDATA blocks for URLs.